docs/firmware-update.md

```diff
@@ -14,6 +16,24 @@ Steps to flash the device:
 
 ![Firmware Update Waiting](/ping-viewer/images/viewer/firmware-update-waiting.gif)
 
+## Manual Firmware Update
+
+1. [Connect](device-manager.md) to the device
+2. Open the Firmware Update menu
+3. Choose 'Manual Update'
+4. Download your desired firmware from the [firmware repository](https://github.com/bluerobotics/ping-firmware/tree/master/ping1d)
+  - Find your desired firmware in the `/ping1d` folder
+  - Click 'Raw' to view the raw document
+  - Right-click the webpage and click 'Save As...' to save the firmware hex file (`.hex`)
```

Comment:
This would be better if they were also an ordered list.

Reply:
done